xml/System.Data.Spatial/DbGeometry.xml

```diff
@@ -1854,8 +1854,8 @@
         <ReturnType>System.Nullable&lt;System.Double&gt;</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>Gets the Y coordinate of this DbGeometry value, if it represents a point. &lt;returns&gt;The Y coordinate value of this geometry value, if it represents a point; otherwise <see langword="null" />.&lt;/returns&gt;</summary>
-        <value>The Y coordinate of this DbGeometry value.</value>
+        <summary>Gets the Y coordinate of this DbGeometry value, if it represents a point.</summary>
+        <value>The Y coordinate value of this geometry value, if it represents a point; otherwise <see langword="null" />.</value>
```

Comment:
```suggestion
        <value>The Y coordinate value of this geometry value, if it represents a point; otherwise, <see langword="null" />.</value>
```